README.md

```diff
@@ -138,6 +138,84 @@ object Main extends App {
 }
 ```
 
+##### Document Streaming
+
+Fauna supports document streaming, where changes to a streamed document are pushed to all clients subscribing to that document.
+
+The following sections provide examples for managing streams with Flow or Monix, and
+assume that you have already created a `FaunaClient`.
+
+###### Flow subscriber
+
+It is possible to use the `java.util.concurrent.Flow` API directly by binding a `Subscriber` manually.
+
+In the example below, we are capturing the 4 first messages:
+
+```scala
+// docRef is a reference to the document for which we want to stream updates.
+// You can acquire a document reference with a query like the following, but it
+// needs to work with the documents that you have.
+// val docRef = client.query(Get(Ref(Collection("scoreboards"), "123")))
```

Comment:
If you do a Get(..), what you get back is the document itself, not its ref. In fact, the following `client.strea(docRef)` call should not work because you're trying to initiate a stream with an object.

The appropriate example would be just:

```scala
val docRef = Ref(Collection("scoreboards"), "123")
```